Packs/Akamai_WAF/ReleaseNotes/2_0_3.md

```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Akamai WAF
+- %%UPDATE_RN%%
```

Comment:
Please add a meaningful update note

Reply:
For some reason, the update description I added while submitting the PR is disappeared. I just added a new one.